Compilers/Linux/gcc.ey

```diff
@@ -12,7 +12,7 @@ Make-Vars:
   cc: gcc
   cxx: g++
   cppflags:
-  cxxflags: -fdiagnostics-color=always
+  cxxflags: -fdiagnostics-color=never
```

Comment:
I don't think this change should be pushed either, cause it affects `emake` afaik.